tox.ini

```diff
@@ -1,20 +1,14 @@
 [tox]
 envlist =
-    py27-dj{13,14,15,16,17,18,19},
-    py34-dj{15,16,17,18,19}
-    py35-dj{17,18,19}
-    py36-dj{17,18,19,22}
-    py37-dj{17,18,19,22}
+    py37-dj{22,32}
+    py38-dj{22,32,40}
+    py39-dj{22,32,40}
+    py310-dj{32,40}
```

Comment:
These are the currently supported Django versions, with compatible Python versions, according to https://www.djangoproject.com/download/